payments-core/src/main/java/com/stripe/android/exception/StripeException.kt

```diff
@@ -43,7 +44,8 @@ abstract class StripeException(
             message == ex.message
     }
 
-    internal companion object {
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    companion object {
         fun create(throwable: Throwable): StripeException {
```

Comment:
Instead of making this public, I think `PaymentFlowResult.Unvalidated#exception` should just be a `Throwable`, and we can transform it to `StripeException` internally.

Reply:
sure, will change this in a follow up